app/experimenter/experiments/tests/test_changelog_utils.py

```diff
@@ -35,24 +35,25 @@ def test_outputs_expected_schema_for_empty_experiment(self):
         self.assertEqual(
             data,
             {
-                "_start_date": None,
                 "_end_date": None,
+                "_start_date": None,
                 "application": "",
                 "branches": [],
                 "channel": NimbusExperiment.Channel.NO_CHANNEL,
                 "conclusion_recommendation": None,
                 "countries": [],
                 "feature_configs": [],
-                "firefox_min_version": NimbusExperiment.Version.NO_VERSION,
                 "firefox_max_version": NimbusExperiment.Version.NO_VERSION,
+                "firefox_min_version": NimbusExperiment.Version.NO_VERSION,
                 "hypothesis": NimbusExperiment.HYPOTHESIS_DEFAULT,
                 "is_archived": experiment.is_archived,
+                "is_client_schema_disabled": experiment.is_client_schema_disabled,
```

Comment:
by default it takes the` False` value, I think we should add test case with `True` value too, what do you think @jaredlockhart 🤔

Reply:
Oh yeah I understand your point but that gets huge really fast if you want to try to do the combinatoric values of every field.  We trust that the serializer will do its job and that the field is declared as a `BooleanField` so no I don't think it's necessary.